Modules/CetTest.cmake

```diff
@@ -264,6 +264,35 @@ CetTest
 #
 ########################################################################
 
+########################################################################
+# cet_test_prepend_env: prepend directories to the specified environment
+#                       variable all tests here specified.
+#
+# Usage: cet_test_prepend_env(<env> [<options] [<dir>+])
+#
+####################################
+# Options:
+#
+# REMOVE_DUPLICATES
+#   Remove any duplicate directory entries that may appear in the
+#   directories specified by the user.  For a given duplicate, only
+#   the first entry will be kept.
+#
+####################################
+# Notes:
+#
+# * <dir> is a directory string or generator expression whose
+# * evaluation will be prepended to the specified environment
+# * variable.  If no <dir> argument is given, then
+# * cet_test_prepend_env() is a NOP.
+#
+# * If cet_test_prepend_env() is called in a directory to set the
+#   environment for tests then that will be propagated to tests
+#   defined in subdirectories unless cet_test_env(CLEAR ...) is
+#   invoked in that directory.
+#
+########################################################################
+
```

Comment:
I should have removed the comment-based documentation when I completed the Sphinx documentation: you don't need to duplicate your doc here, sorry.

Reply:
No problem, I'll remove mine.